gobblin-metrics-libs/gobblin-metrics-base/src/main/java/org/apache/gobblin/metrics/event/EventSubmitter.java

```diff
@@ -36,9 +36,7 @@
  *   Instances of this class are immutable. Calling set* methods returns a copy of the calling instance.
  * </p>
  *
- * @deprecated Use {@link GobblinEventBuilder}
  */
-@Deprecated
 public class EventSubmitter {
```

Comment:
Deprecate `Builder` and provide new constructor
`EventSubmitter(MetricContext context)`

Reply:
Done